src/controller/CurrentFabricRemover.cpp

```diff
@@ -37,10 +37,14 @@ CHIP_ERROR CurrentFabricRemover::RemoveCurrentFabric(NodeId remoteNodeId, Callba
 CHIP_ERROR CurrentFabricRemover::ReadCurrentFabricIndex(Messaging::ExchangeManager & exchangeMgr,
                                                         const SessionHandle & sessionHandle)
 {
+#if CHIP_CONFIG_ENABLE_READ_CLIENT
```

Comment:
This whole class should not be built if !CHIP_CONFIG_ENABLE_READ_CLIENT, because it can't do anything that way....  Followup is fine for that, but please clean it up.

Reply:
Removed this file from the sources list in gn file, with other two cpp files